src/rules/javascript/only-prefs-in-defaults.js

```diff
@@ -1,12 +1,15 @@
+import path from 'path';
+
 import { ONLY_PREFS_IN_DEFAULTS } from 'messages/javascript';
 import { getRootExpression } from 'utils';
 
+
 export default {
   create(context) {
-    var filename = context.getFilename();
+    var relPath = path.relative(process.cwd(), context.getFilename());
```

Comment:
Yeah, I want to make this configurable and maybe forward the actual package path, not sure yet, see https://github.com/mozilla/addons-linter/pull/1140#discussion_r102708372 on that.

Generally, this is how eslint does things anyway so I don't expect there to be any major problems.